README.md

```diff
@@ -7,7 +7,7 @@
 sbt-conductr-sandbox aims to support the running of a Docker-based ConductR cluster in the context of a build. The cluster can then be used to assist you in order to verify that endpoint declarations and other aspects of a bundle configuration are correct. The general idea is that this plugin will also support you when building your project on CI so that you may automatically verify that it runs on ConductR.
 
 ## Prerequisites
-- [Docker](https://www.docker.com/)
+- [Docker 1.8.x or above](https://www.docker.com/)
```

Comment:
Is there a reason for this version in particular? I think we stipulate a version (1.5?) in the ConductR doc already... Ubuntu LTS users get 1.5 via apt I think...